spec/services/code_climate/update_all_projects/code_climate_update_all_projects_service_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+describe CodeClimate::UpdateAllProjectsService do
+  subject { CodeClimate::UpdateAllProjectsService }
+  let(:update_all_projects_code_climate_info) { subject.call }
+
+  it 'with no projects does no fail' do
+    expect { update_all_projects_code_climate_info }.not_to raise_error
+  end
+
+  context 'with a project' do
+    before do
+      first_project
```

Comment:
You can use `let!` below and remove this block

Reply:
Fixed it. Thanks!